src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -63,7 +64,12 @@ def wait_process(proc, wait_time):
     @staticmethod
     def cleanup_processes():
         """ Clean up cart processes, in case avocado/apricot does not. """
-        stop_processes(["localhost"], "'(crt_launch|orterun)'")
+        error_list = []
+        localhost = socket.gethostname().split(".")[0]
+        result = stop_processes(localhost, "'(crt_launch|orterun)'")
+        if 0 in result and len(result) == 1:
```

Comment:
The `result` is a dictionary of return codes keys with `NodeSet` values identifying the nodes with that return code.  We want to report an error if we get a non-zero return code, like in [this ](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-4417/7/artifact/Functional/job-2021-01-28T06.15-190d96e-cart-cart_selftest_three_node/job.log) example:
```
2021-01-28 06:17:15,634 test             L0239 INFO | *** TEARDOWN called after test completion: elapsed time: 78.4761109352 seconds ***
2021-01-28 06:17:15,634 test             L0242 INFO | Amount of time left in test timeout: 521.523889065 seconds
2021-01-28 06:17:15,634 general_utils    L0603 INFO | Killing any processes on wolf-102vm1 that match: '(crt_launch|orterun)'
2021-01-28 06:17:15,700 output           L0655 DEBUG| Command:
2021-01-28 06:17:15,700 output           L0655 DEBUG|   rc=0; if pgrep --list-full '(crt_launch|orterun)'; then rc=1; sudo pkill '(crt_launch|orterun)'; sleep 5; if pgrep --list-full '(crt_launch|orterun)'; then pkill --signal ABRT '(crt_launch|orterun)'; sleep 1; if pgrep --list-full '(crt_launch|orterun)'; then pkill --signal KILL '(crt_launch|orterun)'; fi; fi; fi; exit $rc
2021-01-28 06:17:15,700 output           L0655 DEBUG| Command return codes:
2021-01-28 06:17:15,700 output           L0655 DEBUG|   [0-2],-,f,l,m,o,v,w: rc=255
2021-01-28 06:17:15,701 output           L0655 DEBUG| Command output:
2021-01-28 06:17:15,701 output           L0655 DEBUG|   1:
2021-01-28 06:17:15,701 output           L0655 DEBUG|     ssh: connect to host 1 port 22: Invalid argument
2021-01-28 06:17:15,701 output           L0655 DEBUG|   l:
2021-01-28 06:17:15,702 output           L0655 DEBUG|     ssh: Could not resolve hostname l: Name or service not known
2021-01-28 06:17:15,702 output           L0655 DEBUG|   m:
2021-01-28 06:17:15,702 output           L0655 DEBUG|     ssh: Could not resolve hostname m: Name or service not known
2021-01-28 06:17:15,702 output           L0655 DEBUG|   f:
2021-01-28 06:17:15,702 output           L0655 DEBUG|     ssh: Could not resolve hostname f: Name or service not known
2021-01-28 06:17:15,703 output           L0655 DEBUG|   w:
2021-01-28 06:17:15,703 output           L0655 DEBUG|     ssh: Could not resolve hostname w: Name or service not known
2021-01-28 06:17:15,703 output           L0655 DEBUG|   2:
2021-01-28 06:17:15,703 output           L0655 DEBUG|     ssh: connect to host 2 port 22: Invalid argument
2021-01-28 06:17:15,703 output           L0655 DEBUG|   o:
2021-01-28 06:17:15,704 output           L0655 DEBUG|     ssh: Could not resolve hostname o: Name or service not known
2021-01-28 06:17:15,704 output           L0655 DEBUG|   0:
2021-01-28 06:17:15,704 output           L0655 DEBUG|     Host key verification failed.
2021-01-28 06:17:15,704 output           L0655 DEBUG|   -:
2021-01-28 06:17:15,704 output           L0655 DEBUG|     ssh: Could not resolve hostname -: Name or service not known
2021-01-28 06:17:15,705 output           L0655 DEBUG|   v:
2021-01-28 06:17:15,705 output           L0655 DEBUG|     ssh: Could not resolve hostname v: Name or service not known
2021-01-28 06:17:15,706 sysinfo          L0366 DEBUG| Not logging /var/log/messages (lack of permissions)
2021-01-28 06:17:15,706 test             L0779 INFO | PASS 1-./cart/cart_selftest_three_node.py:CartSelfThreeNodeTest.test_cart_selftest;defaultENV-sep-hosts_1-self_np-293d
```
Even though we pass in one node, so the `len(result)` should always be one, it doesn't hurt to use this logic:
```suggestion
        if 0 not in result or len(result) > 1:
```